src/accordion/section.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import uniqueId from '../utils/unique-id.js';
+import styles from './section.css';
+
+const headings = {
+  1: 'h1',
+  2: 'h2',
+  3: 'h3',
+  4: 'h4',
+  5: 'h5',
+  6: 'h6',
+};
+
+export default class Section extends React.Component {
+  constructor() {
+    super();
+    this.accessibleId = uniqueId();
+  }
+
+  render() {
+    const {
+      cannotClose,
+      children,
+      cssContainer,
+      cssHeading,
+      cssPanel,
+      cssTrigger,
+      headingLevel,
+      onClick,
+      open,
+      title,
+    } = this.props;
+    const titleVal = typeof title === 'function' ? title({ open }) : title;
+    const Heading = headings[headingLevel];
+
+    if (!Heading) {
+      throw new Error('Accordion must have a headingLevel between 1 and 6');
+    }
```

Comment:
I'm not a huge fan of runtime checks. But in this case, it's super important from an accessibility standpoint that we get the correct "heading" level. Figured it was worth it here to break fast and loud if someone doesn't set that up with something valid.

Reply:
^ agreed